client/account-recovery/components/account-recovery-error-message/index.jsx

```diff
@@ -16,6 +16,12 @@ class AccountRecoveryErrorMessage extends Component {
 			case 'RestInvalidKeyError':
 				return translate( "We've failed to validate with the given code. " +
 					'Please double check if the code is correct.' );
+
+			// The error messages from our backend are quite descriptive in this case. e.g.
+			// "You've recently used this password", "Password must be at least 6 characeters.", etc.
+			// Thus I'm using it directly here.
+			case 'RestBadPasswordError':
+				return error.message;
```

Comment:
Sure things. I've documented it here: https://github.com/Automattic/wp-calypso/issues/14073